pkgs/development/compilers/rust/binary.nix

```diff
@@ -19,7 +19,7 @@ in
 
 rec {
   rustc = stdenv.mkDerivation {
-    name = "rustc-${versionType}-${version}";
+    pname = "rustc-${versionType}";
 
```

Comment:
There is another one in this file?

https://github.com/NixOS/nixpkgs/blob/22280e4ab21ae607187da3bdc0ddd49b7147bc1c/pkgs/development/compilers/rust/binary.nix#L73-L74

Reply:
thanks for noticing. should be fixed